### PR TITLE
Enable Digital Pack customers to log in to /manage page

### PR DESCRIPTION
**Pull request by paulbrown1982**
- Enabled Digital Pack customers to log in to manage page, and see how long is left on their free trial.
- Tweaked some copy.
- Renamed suspend.scala.html to delivery.scala.html to match the product naming of the other pages. Likewise success.scala.html.
- Tidied up AccountManagement.scala a bit [understatement 🎺 ].

Requires: https://github.com/guardian/membership-common/pull/428

Whilst on free trial:
screen shot 2017-02-15 at 11 30 03

Once free trial is over:
screen shot 2017-02-15 at 13 05 18

cc @jacobwinch @pvighi @johnduffell @AWare

### CODE REVIEW

File: app/views/account/details.scala.html [hunk 1]
```diff
@@ -19,14 +19,14 @@ <h1 class="suspend-header__title">Manage your Guardian subscription</h1>
...
-                    <li>Voucher subscribers can also see when they next book will be delivered</li>
-                    <li>Guardian Weekly subscribers can also renew their 1/2/3-year plans</li>
+                    <li>Digital Pack subscribers can see when their free trial ends</li>
+                    <li>Home Delivery subscribers can create holiday suspensions</li>
+                    <li>Voucher subscribers can see their billing schedule</li>
+                    <li>Guardian Weekly subscribers can renew their 1/2/3-year plans onto 1-year or recurring plans</li>
                 </ul>
                 @*
-                <h4>Coming soon:</h4>
+                <h4>Coming soon?:</h4>
```
Comment (pvighi) [inline app/views/account/details.scala.html]: is this the correct text?

**Comment by pvighi** [comment]
looks good 👍

File: app/views/account/details.scala.html (hunk 1 above)
Comment (paulbrown1982) [inline app/views/account/details.scala.html]: It's hidden behind a template comment.

File: app/controllers/AccountManagement.scala [hunk 2]
```diff
@@ -37,25 +37,25 @@ object SessionSubscription {
...
+  def set(result: Result, sub: Subscription[Paid]): Result =
     result.withSession(
       SUBSCRIPTION_SESSION_KEY -> sub.name.get
     )
 
   def clear(result: Result)(implicit request: Request[AnyContent]): Result =
     result.withSession(request.session - SUBSCRIPTION_SESSION_KEY)
 
-  def subscriptionFromRequest(implicit request: Request[_]): Future[Option[Subscription[SubscriptionPlan.PaperPlan]]] = {
+  def subscriptionFromRequest(implicit request: Request[_]): Future[Option[Subscription[Paid]]] = {
```
Comment (johnduffell) [inline app/controllers/AccountManagement.scala]: so now people can manage digipack and membership now
or perhaps the product should be ContentSubscripton if we only want to pull in digipack

File: app/views/account/delivery.scala.html [hunk 3]
```diff
@@ -34,7 +34,7 @@ <h1 class="suspend-header__title">Cancel your papers while you're away</h1>
                     You can use this service to temporarily suspend delivery of your newspapers.
                 </p>
                 <p>
-                    Please give us at least 5 days notice, and we will reduce the first payment after you go away by the value of the cancelled papers, including delivery.
+                    Please give us at least 5 days notice, and we will reduce the next payment after you go away by the value of the cancelled papers, including delivery.
```
Comment (johnduffell) [inline app/views/account/delivery.scala.html]: if all goes to plan..!

File: app/views/account/details.scala.html (hunk 1 above)
Comment (johnduffell) [inline app/views/account/details.scala.html]: we left this here as a kind of TODO list...

**Comment by johnduffell** [comment]
looks great, good work 👍

**Pull request merged**